site/content/docs/01-component-format.md

```diff
@@ -95,7 +95,7 @@ To change component state and trigger a re-render, just assign to a locally decl
 
 Update expressions (`count += 1`) and property assignments (`obj.x = y`) have the same effect.
 
-Because Svelte's reactivity is based on assignments, using array methods like `.push()` and `.splice()` won't automatically trigger updates. Options for getting around this can be found in the [tutorial](tutorial/updating-arrays-and-objects).
+Because Svelte's reactivity is based on assignments, using array methods like `.push()` and `.splice()` won't automatically trigger updates, to trigger the update a subsequent assignment is requirred. This and more details can also be found in the [tutorial](tutorial/updating-arrays-and-objects).
```

Comment:
```suggestion
Because Svelte's reactivity is based on assignments, using array methods like `.push()` and `.splice()` won't automatically trigger updates. To trigger the update a subsequent assignment is required. This and more details can also be found in the [tutorial](tutorial/updating-arrays-and-objects).
```